### DISCUSSION TITLE
[1.9.2] 2024-02-18

### DISCUSSION THREAD

**Discussion by darthcloud** [darthcloud/BlueRetro#914] [General]
# Changelog

## New features
* Add support for PS2/3 pressure buttons
* Persistent customizable default config via NVS variables
   See [docs](https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-NVS-Config-Generator)

## Bug fixes
* Fix polarity of the RY axis for generic wireless devices (Thanks @hworld)
* Fix mapping regression for some 8bitdo controller
* Fix sign of 8 bits logical min & max in HID parser
* Fix Switch LED getting change on HW2 when using N64 acc toggle functions

# Firmware is available in 2 different flavor
## 1. Regular FW as before retroactively renamed HW1 (Typically for external adapter)
Use firmwares from HW1 archive for:
* [WillConsole's](https://twitter.com/WillConsole) PSUnoRetro
* All [Humble Bazooka's](https://twitter.com/humblebazooka) BT dongle
* [RetroTime's](https://twitter.com/bixxewoscht) N64 dongle
* [RetroOnyx's](https://twitter.com/mellott124) VB dongle
* [GrechTech's](https://twitter.com/GrechTech) RetroRosetta
* any DIY adapter you made base on the original BlueRetro spec
* most likely anything from aliexpress

## 2. HW2 FW that support power management and port detection (Typically for internal install)
Use firmwares from HW2 archive for:
* [Laser Bear Industries's](https://twitter.com/collingall) GameCube BlueRetro Internal Adapter
* any DIY internal install you made base on the HW2 spec

## [OTA Firmware update page](https://blueretro.io/ota.html) has protection to prevent cross flashing between HW1 & HW2 firmware.

# Download latest firmware archives from https://darthcloud.itch.io/blueretro

<hr /><em>This discussion was created from the release <a href='https://github.com/darthcloud/BlueRetro/releases/tag/v1.9.2'>[1.9.2] 2024-02-18</a>.</em>

**Comment by golimonkey** [comment]
This release fixed issues with reconnecting my fake DS4s to blueretro. (before this update they would only connect just after pairing so I had to pair them every time, otherwise reconnecting would just fail) TY <3
